packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -63,17 +63,19 @@ function renderLogMessage(
 ) {
   const needsHighlighter =
     highlights && highlights.length > 0 && highlights[0] && highlights[0].length > 0 && entry.length < MAX_CHARACTERS;
-  if (needsHighlighter) {
+  const searchWords = highlights ?? [];
+  if (hasAnsi) {
+    const highlight = needsHighlighter ? {searchWords, highlightClassName} : undefined;
```

Comment:
i'm sorry, the change i requested made the linter angry 😢 , i think we need to put spaces inside the curly-braces... to be sure, please run `yarn lint`, then fix the problem and run `yarn lint` again to be sure the problems is solved. thanks!

Reply:
OK, no problem, should be sorted now.